zenodo/modules/records/templates/zenodo_records/macros/admin.html

```diff
@@ -67,7 +73,12 @@
 {%- endmacro %}
 
 {%- macro safelist_action(user, action) %}
-  {{user is safelisted_user}}
+  {% if user is safelisted_user %}
```

Comment:
As discussed IRL, if you apply the [`uselist=False` parameter](https://github.com/inveniosoftware/invenio-accounts/blob/8d993b7f6239891e6d645a3857ba5ca492849974/invenio_accounts/models.py#L137) on the `db.relationship` of the model, you could instead do something like:

```suggestion
  {% if user.safelist %}
```

Reply:
Don't remember why we decided to go with this, ofc it works, but a reminder would be appreciated :D